index.js

```diff
@@ -17,7 +19,14 @@ module.exports = function (str, filepath) {
 
 	return pify(mkdirp, Promise)(path.dirname(fullpath))
 		.then(function () {
-			return pify(fs.writeFile, Promise)(fullpath, str);
+			const pipeStream = () => {
```

Comment:
Darn it. I almost made it through the PR without doing that. :)

Thanks for catching that! I wish something in `npm test` would tell me about this.